File: tasks/Algorithms_Problem Solving/palindrome.py
"""Program for palindrome number"""


def palin(num):
    """function for checking palindrome"""
    num1 = num[::-1]
    if num == num1:
        print("Number is palindrome")
    else:
        print("Number is not palindrome")


n = input("Enter a number\n")
palin(n)

File: JavaPractise/Demo/src/Palindrome.java
import java.util.Scanner;

public class Palindrome {

	public static void main(String[] args) {
		Scanner sc=new Scanner(System.in);
		System.out.println("Enter the number:");
       String num=sc.nextLine();
       String rev= "";
       for(int i=num.length()-1;i>=0;i--) {
    	   rev+=num.charAt(i);
       }
       if(rev.equals(num)) {
    	   System.out.println("Palindrome");
       }
       else {
    	   System.out.println("Not  a palindrome");
       }
	}

}
